automatic/sysinternals/Readme.md

```diff
@@ -9,13 +9,14 @@ It does not contain non-troubleshooting tools like the BSOD Screen Saver or NotM
 
 - `/InstallDir` - Installation directory, by default Chocolatey tools directory.
 - `/InstallationPath` - the same as `InstallDir`
+- `/NoStartMenu` - Set to true to prevent installation of a menu item.
```

Comment:
This should be inverted to `/StartMenu` (and you don't need to set it to true, just passing that in is enough).